sum.c

```diff
@@ -0,0 +1,16 @@
+//program to find the sum inclusive of the start and the end no. mentioned
+/*
+Created on november 8, 2018
+@author: sumangala rao
+*/
+#include<stdio.h>
+  int main()
+   {
+     int n1,n2,i,sum=0;         //initialization
```

Comment:
Place the comments regarding the line above the line.

Comments are placed above a block. Blocks should be separated by a blank line.

In this case. A line is a block and lines should be separated by a blank line.